src/NServiceBus.Core/Settings/SettingsHolder.cs

```diff
@@ -201,6 +201,11 @@ public void Set(string key, object value)
         /// </summary>
         /// <typeparam name="T">The type to use as a key for storing the setting.</typeparam>
         /// <param name="value">The setting value.</param>
+        [ObsoleteEx(
+            Message = "Deplecated method. Instead, use Set<T>(T value).",
```

Comment:
Spelling: Deplecated

"Instead, use Set<T>(T value)." is not necessary. If you look at the approval file you'll see that "Use `SetDefault<T>(T value)` is automatically generated and causes the message to be duplicated.